src/app/zap-templates/zcl/data-model/chip/thread-border-router-management-cluster.xml

```diff
@@ -0,0 +1,139 @@
+<?xml version="1.0"?>
+<!--
+Copyright (c) 2024 Project CHIP Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<configurator>
+  <domain name="CHIP"/>
+
+  <bitmap name="Feature" type="bitmap32">
+    <cluster code="0x0452"/>
+    <field name="PANChange" mask="0x1"/>
+  </bitmap>
+
+  <enum name="RoutingRoleEnum" type="enum8">
+    <cluster code="0x0452"/>
+    <item name="Unspecified" value="0x00"/>
+    <item name="Unassigned" value="0x01"/>
+    <item name="SleepyEndDevice" value="0x02"/>
+    <item name="EndDevice" value="0x03"/>
+    <item name="REED" value="0x04"/>
+    <item name="Router" value="0x05"/>
+    <item name="Leader" value="0x06"/>
+  </enum>
+
+  <struct name="NeiborTableStruct">
```

Comment:
I really hope this does not match the spec, which again does not seem to exist.

Reply:
Removed this struct